faqs.html

```diff
@@ -5,29 +5,41 @@
 <div class="container white-text">
   <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
   <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <div class="card-content"> <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p>As passionate we are about cybersecurity, safety is our number one priority. We sincerly hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, it has been determined that <strong>CUCTF 2020 will be a completely virtual event on October 3rd. </strong> We refuse to be setback from this announcement, and will use this oppertunity to: </p>
+            <ul class="browser-default">
+                <li>Allow more universities to participate by increasing our CTF infrastructure </li>
+                <li>Present opening and closing ceremonies via Zoom</li>
+                <li>Utilize Slack or Discord to answer questions, push notifications, and maintain a connection with participants</li>
+            </ul></div></div>
```

Comment:
Same as last PR: Inner `<div>` (and especially `card-content` since this isn't a card) does nothing here so just keep it all in the `collapsible-body` classed `<div>`.

```suggestion
      <span class="card-title clem"><i class="material-icons">error_outline</i></span>
        <p>As passionate we are about cybersecurity, safety is our number one priority. We sincerly hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, it has been determined that <strong>CUCTF 2020 will be a completely virtual event on October 3rd. </strong> We refuse to be setback from this announcement, and will use this oppertunity to: </p>
            <ul class="browser-default">
                <li>Allow more universities to participate by increasing our CTF infrastructure </li>
                <li>Present opening and closing ceremonies via Zoom</li>
                <li>Utilize Slack or Discord to answer questions, push notifications, and maintain a connection with participants</li>
            </ul>
          </div>
```

Reply:
You are correct, I didn't notice that the Div was still there after rebranching